src/server/server.js

```diff
@@ -27,10 +27,10 @@ const schema = Type.Object({
     description: 'path to directory where data is stored',
     default: DEFAULT_STORAGE,
   }),
-  ROOT_KEY: Type.Optional(
-    Type.String({
-      description:
-        'hex-encoded 16-byte random secret key, used for server keypairs',
+  ALLOWED_PROJECTS: Type.Optional(
```

Comment:
Nit: I think a name like `MAXIMUM_ALLOWED_PROJECTS` is clearer. I initially thought this was an array of project IDs or something, based on the name. (Probably also means updating the `allowedProjects` identifier elsewhere in the code.)